### ISSUE TITLE
Add ability to pass arguments to tasks

### ISSUE THREAD

**Issue by agentmilindu** [leopardslab/Dunner#3]
When the `.dunner.yaml` is as follows, users should be able to run `Dunner status prod` where `prod` is an argument to the task `status`

```
status:
- name: 'mesosphere/aws-cli'
  command: ['aws', 'elasticbeanstalk describe-events --environment-name $1'] 
  # This uses args passed to the task, `$1` means first arg
```

**Comment by VibhorCodecianGupta** [comment]
Working on this

**Comment by agentmilindu** [comment]
Fixed by #47
